en/docs/build-and-run/build-and-run.md

```diff
@@ -1,9 +1,16 @@
 # Build and run the program
 
-Follow the steps below to create a sample Ballerina program in VS Code.
+Follow the steps below to create a sample Ballerina program in VS Code. 
 
-<img src="https://github.com/wso2/ballerina-plugin-vscode/blob/main/resources/images/running-your-program.gif?raw=true" width="100%" />
+<img src="../img/running-your-program.gif" width="100%" />
 
+</br>
+
+<details open>
+  <summary>Code View</summary>
```

Comment:
Why do we need expandables for these? IMO, level 2 headings for Code View and Diagram View will do.

Build and run the program
- Use the Code View
- Use the Diagram View